initramfs/bootscript

```diff
@@ -30,14 +37,62 @@ tar -xvf - -C "$TMPDIR" \
 STARTUP="$TMPDIR/startup.sh"
 KEY="$TMPDIR/rootfs.key"
 
-if [ -x "$STARTUP" ]; then
+if [[ -x $STARTUP ]]; then
 	# measure the startup script before doing anything else
 	warn "$STARTUP: measuring"
 	tpm2 pcrevent 14 "$STARTUP"
 	warn "$STARTUP: executing"
+	# shellcheck disable=SC1090
 	. "$STARTUP" || die "startup failed"
 fi
 
+# Authenticated decryption; see functions.sh
```

Comment:
can we reuse the one from `functions.sh`? We should have it available to the bootscript.

Reply:
OK, I wasn't sure.

Reply:
Fixed.